src/Home.js

```diff
@@ -39,15 +41,35 @@ class Home extends Component {
     activity: PropTypes.bool.isRequired,
     history: PropTypes.shape({ length: PropTypes.number.isRequired })
       .isRequired,
+    searchQuery: PropTypes.string.isRequired,
   };
 
-  componentDidMount() {
+  state = {
+    // eslint-disable-next-line react/destructuring-assignment
+    filteredSpaces: this.props.spaces,
+  };
+
+  async componentDidMount() {
     const { dispatchGetSpaces } = this.props;
-    dispatchGetSpaces();
+    await dispatchGetSpaces();
```

Comment:
Do we need to add the await here? I don't think it has any effect. In fact, we can probably remove the `async` from the function. Let me know if that is not the case.